### PR TITLE
[en]1.25 Rewrite implementation-details.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Remove some unnecessary comments
Preview of updated page:
https://deploy-preview-36238--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/implementation-details/

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/implementation-details.md [hunk 1]
```diff
@@ -409,12 +409,11 @@ Please note that:
 As soon as the control plane is available, kubeadm executes following actions:
 
 - Labels the node as control-plane with `node-role.kubernetes.io/control-plane=""`
-<<<<<<< HEAD
 - Taints the node with `node-role.kubernetes.io/control-plane:NoSchedule`
 
-Please note that the phase to mark the control-plane phase phase can be invoked
+Please note that the phase to mark the control-plane  phase can be invoked
```
Comment (reylejano) [inline content/en/docs/reference/setup-tools/kubeadm/implementation-details.md:414]: nit: single space between words
```suggestion
Please note that the phase to mark the control-plane phase can be invoked
```

**Comment by reylejano** [comment]
Preview of updated page:
https://deploy-preview-36238--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/implementation-details/

**Comment by reylejano** [comment]
Changes look good and improve the page
/lgtm

File: content/en/docs/reference/setup-tools/kubeadm/implementation-details.md (hunk 1 above)
Comment (yanrongshi) [reply content/en/docs/reference/setup-tools/kubeadm/implementation-details.md]: thanks for the reminder

File: content/en/docs/reference/setup-tools/kubeadm/implementation-details.md [hunk 2]
```diff
@@ -446,8 +445,8 @@ Please note that:
 
 `kubeadm init` create a first bootstrap token, either generated automatically or provided by the
 user with the `--token` flag; as documented in bootstrap token specification, token should be
-saved as secrets with name `bootstrap-token-<token-id>` under `kube-system` namespace.  Please
-note that:
+saved as secrets with name `bootstrap-token-<token-id>` under `kube-system` namespace.
+Please note that:
```
Comment (a-mccarthy) [inline content/en/docs/reference/setup-tools/kubeadm/implementation-details.md:449]: nit: I think you need one more new line space here to get this to appear on the next line, similar to the other "Please note that:" sections on the page. Right now it's still showing in the preview on the same line as the paragraph above. 

Its a small thing, the changes all look good

**Comment by a-mccarthy** [comment]
did a quick review here, and the changes look good!
/lgtm

File: content/en/docs/reference/setup-tools/kubeadm/implementation-details.md (hunk 2 above)
Comment (yanrongshi) [reply content/en/docs/reference/setup-tools/kubeadm/implementation-details.md:450]: Good suggestion, I have updated, please review

**Review by a-mccarthy — COMMENTED** [review]
/lgtm

**Comment by reylejano** [comment]
There's a [second lgtm ](https://github.com/kubernetes/website/pull/36238#pullrequestreview-1085875842)
/approve

**Pull request merged**